BookLibrary-cdi/src/main/java/org/booklibrary/app/manager/exceptions/EntityManagerException.java

```diff
@@ -0,0 +1,31 @@
+package org.booklibrary.app.manager.exceptions;
+
+import javax.ejb.ApplicationException;
+
+/**
+ * Wrap all persistence exceptions in checked exception.
+ * Throwing this exception will perform transaction rollback
+ */
+@ApplicationException(rollback = true)
+public class EntityManagerException extends Exception{
```

Comment:
This exception wrap exceptions from persistence layer and make them checked also perform ejb transaction rollback after throwing